en/frames_rover/traxxas_stampede.md

```diff
@@ -0,0 +1,138 @@
+
+The assembly consists of a wooden frame on which all the autopilot parts were attached. Tests showed that a better vibration insulation should be used, especially for the Pixhawk and the Flow module.
+
+![Stampede Chassis](../../assets/airframes/rover/traxxas_stampede_vxl/stampede_chassis.jpg)
+
+![Wooden Panel Top](../../assets/airframes/rover/traxxas_stampede_vxl/panel_top.jpg)
+
+![Wooden Panel Bottom](../../assets/airframes/rover/traxxas_stampede_vxl/panel_bottom.jpg)
+
+![Traxxas Stampede Final Assembly](../../assets/airframes/rover/traxxas_stampede_vxl/final_assembly.jpg)
+
+![Side View Final Assembly](../../assets/airframes/rover/traxxas_stampede_vxl/final_side.jpg)
+
+![Wodden panel fixture](../../assets/airframes/rover/traxxas_stampede_vxl/mounting_detail.jpg)
+
+For this particular mounting we chose to use the clip supplied with the rover to attach the upper plate. For this, two supports were 3D printed. The CAD files are provided [here](https://github.com/PX4/Devguide/raw/master/assets/airframes/experimental/stampede/plane_holders.zip).
+
+> **Warning** It is **HIGHLY RECOMMENDED** to set the ESC in training mode (see Traxxas Stampede Manual) so to reduce the power to 50%.
+
+
+## Output connections
+
+| PWM Output | Rate | Actuator |
+| -- | -- | -- |
+| MAIN1 | 50 Hz | Not connected |
+| MAIN2 | 50 Hz | Steering servo |
+| MAIN3 | 50 Hz | Not Connected |
+| MAIN4 | 50 Hz | ESC input |
+| MAIN5 | 50 Hz | Not Connected |
+| MAIN6 | 50 Hz | Not Connected |
+| MAIN7 | 50 Hz | Not Connected |
+| MAIN8 | 50 Hz | Not Connected |
+
+
+## Configuration
+
+Rovers are configured using *QGroundControl* in the same way as any other vehicle.
+
+The main rover-specific configuration is setting the correct frame:
+1. Switch to the [Basic Configuration](../config/README.md) section in *QGroundControl*
+1. Select the [Airframe](../config/airframe.md) tab. 
+1. Scroll down the list to find the **Rover** icon. 
+1. Choose **Traxxas stampede vxl 2wd** from the drop down list.
+
+![Select Airframe](../../images/qgc/setup/airframe_px4_rover_traxxas_stampede_vxl_2wd.jpg)
+
+
+## Usage
+
+At the current time, PX4 only supports MISSION and MANUAL modes when a RC remote is connected. To use the mission mode, first upload a new mission to the vehicle with QGC. Then, BEFORE ARMING, select `MISSION` and then arm. 
+
+> **Warning** It is **_VERY IMPORTANT_** to do a mission composed **_ONLY_** of normal waypoints (i.e. **_NO TAKEOFF WAYPOINTS_**) and it is crucial to **_SET THE WAYPOINT HEIGHT OF EVERY WAYPOINT TO 0_** for a correct execution. Failing to do so will cause the rover to continuously spin around a waypoint.
+
+
+A correct mission setup looks as follows
+
+![mission](../../assets/airframes/rover/traxxas_stampede_vxl/correct_mission.jpg)
+ 
+Since at the current state of development there is no obstacle avoidance, it is highly recommended to test missions in the simulator first to get acquainted with the software usage.
+
+
+<!-- 
+
+## Parameters
```

Comment:
@dagar I'm going to merge this because otherwise this PR becomes unmanagable. Would appreciate your comment on https://github.com/PX4/px4_user_guide/pull/52/files#r133645873 though.